src/screens/MainScreen/index.js

```diff
@@ -18,16 +20,44 @@ const MainScreen = () => {
     getCalls();
   }, [currentUser]);
 
+  let testimo = null;
```

Comment:
If this is a reference to an object, best to use the Hook API useRef. Also a more descriptive name lol :p

Reply:
done.